src/main/resources/com/github/mylibrelab/ui/icons/bold.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><rect id="frame" width="16" height="16" fill="none"/><g data-name="Layer 2"><path fill="#6e6e6e" fill-rule="evenodd" d="M14 7H4.54l2.68-2.68-1.27-1.27-3.68 3.68L1 8l1 1 .27.27 3.68 3.68 1.27-1.27L4.55 9H14V7z" data-name="Layer 1"/></g></svg>
```

Comment:
Why is this an arrow.

Reply:
Fair questions. No idea how that happened.